webcompat/static/config_data/labels.json

```diff
@@ -0,0 +1,182 @@
+[
+  {
+    "url": "https://api.github.com/repos/webcompat/web-bugs/labels/status-duplicate",
+    "name": "status-duplicate",
+    "color": "cccccc"
```

Comment:
Well, it's not exactly hand-written - just downloaded from GitHub :-p

Anyway. To build this file on startup, I should add code to webcompat/__init__.py - right? Or run.py?